randovania/games/planets_zebeth/exporter/patch_data_factory.py

```diff
@@ -1,12 +1,136 @@
+        starting_memo = None
+        extra_starting = item_names.additional_starting_equipment(self.configuration, self.game, self.patches)
+        if extra_starting:
+            starting_memo = ", ".join(extra_starting)
+        return starting_memo
+
+    def _create_starting_location(self) -> dict:
+        return {
+            "x": self.game.region_list.node_by_identifier(self.patches.starting_location).extra["global_x"],
+            "y": self.game.region_list.node_by_identifier(self.patches.starting_location).extra["global_y"],
+        }
+
+    def _create_hash_dict(self) -> dict:
+        return {
+            "word_hash": self.description.shareable_word_hash,
+            "hash": self.description.shareable_hash,
+            "session_uuid": str(self.players_config.get_own_uuid()),
+        }
+
+    def _create_game_config_dict(self):
+        return {
+            "starting_room": self._create_starting_location(),
+            "seed_identifier": self._create_hash_dict(),
+            "starting_items": self._create_starting_items_dict(),
+            "starting_memo": self._create_starting_memo(),
+        }
+
+    def _create_cosmetics(self) -> dict:
+        c = self.cosmetic_patches
+        return {
+            "show_unexplored_map": c.show_unexplored_map,
+            "room_names_on_hud": c.show_room_names.value,
+        }
+
+    def _get_item_data(self):
+        item_data = json_lib.read_path(
+            RandovaniaGame.METROID_PLANETS_ZEBETH.data_path.joinpath("pickup_database", "item_data.json")
+        )
+
+        item_data["Missiles"] = item_data["Missile Tank"]
+        item_data["EnergyTransferModule"] = item_data["Nothing"]
+        return item_data
+
+    def _create_object_datas(self):
+        pass
+
     def game_enum(self) -> RandovaniaGame:
         return RandovaniaGame.METROID_PLANETS_ZEBETH
 
     def create_data(self) -> dict:
-        return {}
+        db = self.game
+
+        useless_target = PickupTarget(
+            pickup_creator.create_nothing_pickup(db.resource_database, "spr_ITEM_Nothing"),
+            self.players_config.player_index,
+        )
+
+        item_data = self._get_item_data()
+        memo_data = {key: value["text_desc"] for key, value in item_data.items()}
+        memo_data["Energy Tank"] = memo_data["Energy Tank"].format(Energy=100)
+
+        pickup_list = pickup_exporter.export_all_indices(
+            self.patches,
+            useless_target,
+            self.game.region_list,
+            self.rng,
+            self.configuration.pickup_model_style,
+            self.configuration.pickup_model_data_source,
+            exporter=pickup_exporter.create_pickup_exporter(memo_data, self.players_config, self.game_enum()),
+            visual_nothing=pickup_creator.create_visual_nothing(self.game_enum(), "spr_ITEM_Nothing"),
+        )
+
+        return {
+            "seed": self.description.get_seed_for_player(self.players_config.player_index),
```

Comment:
out of curiosity, any reason why this needs to be provided to the patcher? what will it be used for?